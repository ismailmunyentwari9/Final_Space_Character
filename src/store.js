import { configureStore } from '@reduxjs/toolkit';
import charactersReducer from './Redux/finalspace/finalspaceSlice';

const store = configureStore({
  reducer: {
    characters: charactersReducer,
  },
});

export default store;
