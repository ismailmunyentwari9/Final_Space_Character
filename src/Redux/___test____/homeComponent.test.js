import React from 'react';
import { Provider } from 'react-redux';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Home from '../../components/home';
import store from '../../store';

it('matches snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Home />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
