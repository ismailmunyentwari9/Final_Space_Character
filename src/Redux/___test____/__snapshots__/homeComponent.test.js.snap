// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container cards-holder"
>
  <h1>
    Final Space Characters
  </h1>
  <div
    className="input-group mb-3"
  >
    <input
      className="form-control"
      onChange={[Function]}
      placeholder="Search..."
      type="text"
      value=""
    />
    <div
      className="input-group-append"
    >
      <button
        className="btn btn-outline-secondary"
        type="button"
      >
        <i
          className="fas fa-search"
        />
      </button>
    </div>
  </div>
  <div
    className="row"
  />
</div>
`;
